src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -761,6 +793,10 @@ export default {
 			 * @type {HTMLElement}
 			 */
 			this.$emit('closed', element)
+
+			// Return focus to the element that had focus before the sidebar was opened
+			this.elementToReturnFocus?.focus({ focusVisible: true })
```

Comment:
Maybe note that this is only supported in Firefox currently?

Reply:
The recording in the PR was done in Chromium-based browser...

Reply:
Guess MDN is not up-to-date then :(